### DISCUSSION TITLE
Remote level control

### DISCUSSION THREAD

**Discussion by cheng93** [datalust/winston-seq#18] [General]
https://nblumhardt.com/2016/02/remote-level-control-in-serilog-using-seq/

This is done in serilog, however can we do this in winston?

In theory this https://github.com/datalust/seq-logging/pull/23 should give us the functionality, however it looks like it got removed.  Reported here https://github.com/datalust/seq-logging/issues/78

**Comment by liammclennan** [comment]
I'll take a look. Thanks for letting us know. Since you are reporting an issue on `seq-logging` I'll leave that one and close this one.

**Comment by cheng93** [comment]
`seq-logging` gives us the `onRemoteConfigChange` hook. I believe each individual package (i.e winston-seq/pino-seq etc) will need to implement it

**Reply by cheng93** [reply]
I suppose the consumers can implement `onRemoteConfigChange` callback themselves, so maybe not!
